### PR TITLE
resolves #56 remove obsolete gadget

### LINKED ISSUE #56: Remove obsolete renderGadget code from document.html.erb template
https://github.com/neo4j-contrib/training-v2/blob/27180c9df6fc9d8aa30375b2c4801e5ae633a5fa/Courses/_templates/course/document.html.erb#L161-L188

### PR DESCRIPTION
**Pull request by Mogztter**
:fire: **Review and merged with caution** :fire: 

resolves #56

### CODE REVIEW

File: Courses/_templates/course/document.html.erb [hunk 1]
```diff
@@ -122,71 +115,37 @@
...
+  }
 
   pre.cypher-syntax {
     background-color: #eaf1f8;
   }
 </style>
-<script src="https://neo4j.com/online_training/cypher-gadget/libs/underscore.js"></script>
-<script src="https://neo4j.com/online_training/cypher-gadget/libs/jquery.js"></script>
-<script src="https://neo4j.com//online_training/cypher-gadget/libs/jquery-ui.min.js"></script>
-<script src="https://neo4j.com/online_training/cypher-gadget/libs/jquery.modal.min.js"></script>
```
Comment (Mogztter) [inline Courses/_templates/course/document.html.erb:154]: I think we are not using the above scripts and `jquery` is already declared above (https://ajax.googleapis.com/ajax/libs/jquery/2.2.0/jquery.min.js).

As a side note, we are using a more than 3 years old version of JQuery! We should probably remove this dependency.

**Comment by Mogztter** [comment]
Nevermind `renderGadget` function is used in https://neo4j.com/online_training/tasks.js... but all the selectors seem to be absent from the HTML: `#recommendActorsForKeanu`,  `#fiveBusiestActors`...

@ElaineRosenberg @jexp could you please confirm that we can safely remove this code?

**Comment by ElaineRosenberg** [comment]
You can remove renderGadget. It is from much older courseware.

**Comment by Mogztter** [comment]
Ok I will first apply these changes manually on a single page to make sure that it does not have side effects.

**Comment by jexp** [comment]
@Mogztter did your manual test work?

**Comment by Mogztter** [comment]
Yes it's working.
I also removed the `neo4j.com/online_training/tasks.js` script but otherwise it seems to work just fine.

**Comment by Mogztter** [comment]
I've published the following courses with these changes if you want to give it a try:

- https://neo4j.com/graphacademy/online-training/neo4j-administration-dev/
- https://neo4j.com/graphacademy/online-training/applied-algos-dev

**Comment by ElaineRosenberg** [comment]
These dev courses work just fine.

**Pull request merged**